docs/modules/ROOT/pages/references/release_notes.adoc

```diff
@@ -2,6 +2,71 @@
 
 TIP: This page lists notable changes in OpenShift releases which we find important. Reading release notes for you as a service.
 
+== OpenShift 4.19
+
+OpenShift version 4.19 is available since 2025-06-17.
+This version is based on Kubernetes 1.32 and CRI-O 1.32.
+The RHCOS image uses RHEL 9.6 packages.
+Find the release notes in the upstream documentation at https://docs.redhat.com/en/documentation/openshift_container_platform/4.19/html/release_notes/ocp-4-19-release-notes[OpenShift Container Platform 4.19 release notes].
+The https://www.redhat.com/en/blog/red-hat-openshift-419-accelerates-virtualization-and-enterprise-ai-innovation[Red Hat unveils OpenShift 4.19] blog post is also a valuable resource.
+
+Direct authentication with external OIDC identity providers is coming to Tech Preview::
+With this release, you can enable direct integration with an external OpenID Connect (OIDC) identity provider to issue tokens for cluster authentication.
+The built-in OAuth server is bypassed.
++
+For further information, see https://docs.redhat.com/en/documentation/openshift_container_platform/4.19/html-single/authentication_and_authorization/#external-auth[Enabling direct authentication with an external OIDC identity provider].
```

Comment:
IIRC -- historically -- we've not added tech preview features in the release summary, since we usually don't really want to make such features available to customers yet.

Reply:
Fair enough!